packages/backend-core/src/db/couch/DatabaseImpl.ts

```diff
@@ -70,7 +70,15 @@ export class DatabaseImpl implements Database {
     DatabaseImpl.nano = buildNano(couchInfo)
   }
 
-  async exists() {
+  exists(docId?: string) {
+    if (docId === undefined) {
```

Comment:
should this be a `== undefined` to cover null as well just incase?

Reply:
This is the typescript way to support overriding. We will have:
1. db.exists()
2. db.exists(docId)

We should never have null, in this case it should be treated as docId in my opinion

Reply:
Yeah thats fair - I guess it shouldn't be a valid input!